docs_source_files/content/introduction/_index.ja.md

```diff
@@ -9,6 +9,6 @@ weight: 2
 日本語は話せますか？プルリクエストをして翻訳を手伝ってください!
```

Comment:
@miekof could you please remove the notice? The page is already fully translated.

Reply:
Removed notifications.